modules/tracer-otel/src/main/java/org/opensearch/tracing/OTelTracerModulePlugin.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.tracing;
+
+import io.opentelemetry.api.OpenTelemetry;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.common.util.FeatureFlags;
+import org.opensearch.plugins.Plugin;
+import org.opensearch.plugins.TracerPlugin;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.function.Supplier;
+
+import static org.opensearch.common.util.FeatureFlags.TRACER;
+
+/**
+ * Tracer plugin based on Otel
+ */
+public class OTelTracerModulePlugin extends Plugin implements TracerPlugin {
+
+    static final String OTEL_TRACER_NAME = "otel";
+
+    /**
+     * No-args constructor
+     */
+    public OTelTracerModulePlugin() {}
+
+    @Override
+    public Settings additionalSettings() {
+        if (FeatureFlags.isEnabled(TRACER)) {
+            return Settings.builder()
+                // set Otel tracer as default tracer
+                .put(TracerModule.TRACER_DEFAULT_TYPE_SETTING.getKey(), OTEL_TRACER_NAME)
```

Comment:
you will set only if there is no explicit tracer_type provided otherwise, this will override it?

Reply:
It will fail we try to override it.

Reply:
This code is removed now